docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -73,22 +73,23 @@ the RPMs for those projects exist by executing the ``pkg:check_published`` Rake
 
       If you see a message like
       ``Warning:  Unable to generate build-specific YUM cache``, your
-      results are invalid, as connection to `PackageCloud`_ failed.
+      results are invalid, as connection to the `SIMP Download Server`_ failed.
 
-#. Manually verify the appropriate ``simp-doc`` RPM exists at `PackageCloud`_.
+#. Manually verify the appropriate ``simp-doc`` RPM exists on the
+`SIMP Download Server`_.
 
 
 For the external vendor RPMs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* Upload all vendor RPMs to the ``VERSION_Dependencies`` repository in
-  `PackageCloud`_. Any existing RPMs will not be overwritten.
+* Upload all vendor RPMs to the proper vendor repository in the relevant version
+  repository on the `SIMP Download Server`_. Any existing RPMs will not be overwritten.
```

Comment:
Similar comments to the above; if it's just going link outside of the documentation to a top-level FQDN, use the URL for clarity.